Header header
Header image_header
BoundingBox[] bounding_boxes
int64 cam_id
